apps/vr-tests/src/stories/Menu.stories.tsx

```diff
@@ -0,0 +1,172 @@
+import { storiesOf } from '@storybook/react';
+import * as React from 'react';
+import Screener from 'screener-storybook/src/screener';
+import {
+  Menu,
+  MenuTrigger,
+  MenuPopover,
+  MenuList,
+  MenuItem,
+  MenuItemCheckbox,
+  MenuItemRadio,
+  MenuGroup,
+  MenuGroupHeader,
+  MenuDivider,
+} from '@fluentui/react-menu';
+import { Cut20Regular, Edit20Regular, ClipboardPaste20Regular } from '@fluentui/react-icons';
+import { FluentProviderDecorator } from '../utilities/index';
+
+storiesOf('Menu - basic', module)
+  .addDecorator(story => (
+    <Screener
+      steps={new Screener.Steps().hover('[role="menuitem"]').snapshot('hover menuitem').end()}
+    >
+      {story()}
+    </Screener>
+  ))
+  .addDecorator(FluentProviderDecorator)
+  .addStory('default', () => (
+    <Menu open>
+      <MenuTrigger>
+        <button>Toggle menu</button>
+      </MenuTrigger>
+
+      <MenuPopover>
+        <MenuList>
+          <MenuItem icon={<Cut20Regular />}>Cut </MenuItem>
+          <MenuItem icon={<Edit20Regular />}>Edit</MenuItem>
+          <MenuItem icon={<ClipboardPaste20Regular />}>Paste</MenuItem>
+        </MenuList>
+      </MenuPopover>
+    </Menu>
+  ));
+
+storiesOf('Menu - secondary content', module)
+  .addDecorator(story => <Screener>{story()}</Screener>)
```

Comment:
Can you please also add "hover" state to this example? _To ensure that we will catch changes in `secondaryContent`_